tests/libtest.sh

```diff
@@ -542,7 +542,7 @@ skip_one_without_bwrap () {
 }
 
 skip_without_fuse () {
-    fusermount --version >/dev/null 2>&1 || skip "no fusermount"
+    ${FUSERMOUNT} --version >/dev/null 2>&1 || skip "no fusermount"
```

Comment:
Shell nitpick:

```suggestion
    "${FUSERMOUNT}" --version >/dev/null 2>&1 || skip "no fusermount"
```

Reply:
Done :) Though seems like you really want a shellcheck CI job for this. I tried `shellcheck $(find tests -name "*.sh") `  and got way too many warnings.

Reply:
I know :'-(

If I had written the whole test suite myself, I would have wanted it to be shellcheck-clean from the start; but I didn't, and retrofitting defensive shell programming onto existing code is a slow process. All I can realistically do without a huge/unreviewable PR is make it not get any worse, and gradually improve bits of it over time.

Reply:
That's actually a very good rationale! Thanks for the explanation!